### ISSUE TITLE
Is the program abandoned?

### ISSUE THREAD

**Issue by mrkaban** [simonsteele/pn#198]
Thank you very much for the free open source program!

Hello, thank you very much for a good program under a free open source license! Added a program to a Russian-language site about free software, here is the [link](https://xn--90abhbolvbbfgb9aje4m.xn--p1ai/%D1%80%D0%B0%D0%B7%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D1%83/%D1%80%D0%B5%D0%B4%D0%B0%D0%BA%D1%82%D0%BE%D1%80%D1%8B-%D0%BA%D0%BE%D0%B4%D0%B0/programmer-s-notepad-%D1%82%D0%B5%D0%BA%D1%81%D1%82%D0%BE%D0%B2%D1%8B%D0%B9-%D1%80%D0%B5%D0%B4%D0%B0%D0%BA%D1%82%D0%BE%D1%80.html).

Will stable releases be published here?

The project is interesting, so please don't abandon it!

**Comment by datenheim** [comment]
Well, none of the 4 repositories of simonsteele got any update after 2017.
Mangling all forks gives the same outcome - no active fork after 2017.
Effectively I would consider this as abandoned. Fork it, or maybe contact
simonsteele to take the flag.

**Comment by akoppa** [comment]
Very sad this program is abandoned since there are lots of improvements that could be included.

**Comment by tonypdmtr** [comment]
The author shows zero activity since 2017 on his repos.
So, either something sad happened to him or his life took a different course.

However, if he is still around, a short message to let us know future plans (if any) would be nice.

It's a pity because it's the one editor I keep coming back from all others I've tried.
(And the only reason I even try others is because I don't see any future with this one.)

**Comment by simonsteele** [comment]
Thanks for all the nice comments. Currently I have no plans to work on Programmer's Notepad in any major form. I simply don't use Windows any more for anything, and family/work commitments leave me no time to work on the project any more. Since 2017 there have been many changes to the set of available text editors, and while I still think PN had some nice unique characteristics you'd like be better off using VSCode or Sublime for many day-to-day tasks.

**Comment by veganaiZe** [comment]
No!  PN is the best!  VSCode is slow and bloated.  Sublime is too difficult to configure, out of the box.  Programmer's Notepad is the superior choice.  It's just fine as is.

Thanks for your hard work.

**Comment by datenheim** [comment]
Hello simonsteele,
thanks for answering, and for bringing PN2 to the world. It was and still is great with the features as is.

Maybe Notepad++ can be a alternative in good standing, small footprint, regular updates,...
at least for features missing in PN2. The other way round, it is missing the Project Tree feature
of PN2.

**Comment by maettu-this** [comment]
I fully understand the restrictions family/work commitments mean, often struggling myself with [YAT](https://sourceforge.net/projects/y-a-terminal/). Still, I support some of the previous statements, **PN2 is my favorite editor for simple text daily work!**

PS @simonsteele: Maybe you should state the fact of no longer developing PN2 at this github page and http://www.pnotepad.org/, or even advertise the search of a potential successor.
